actions/java-gradle-release/action.yaml

```diff
@@ -70,33 +75,27 @@ runs:
         gradle-cache: ${{ inputs.gradle-cache }}
 
     - name: Bump version
-      id: evaluate-version
-      run: |
-        old_version=$(./gradlew properties -q | grep "^version:" | awk '{print $2}' | tr -d '[:space:]' | xargs)
-        if [[ "${{ inputs.release-type }}" == "patch" ]]; then
-            release_version="${old_version%-*}"
-        else
-            release_version=$(semver bump "${{ inputs.release-type }}" "${old_version}")
-        fi
-        echo "release-version=$release_version" >> "$GITHUB_OUTPUT"
-      shell: bash
-
-    - name: Create release
-      run: ./gradlew release -x test -Prelease.useAutomaticVersion=true -Prelease.releaseVersion=${{ steps.evaluate-version.outputs.release-version }} -PdisablePushToRemote=true
-      shell: bash
-      working-directory: ${{ inputs.working-directory }}   
+      id: bump-version
+      uses: bakdata/ci-templates/actions/bump-version@1.32.0
+      with:
+        release-type: ${{ inputs.release-type }}
+        working-directory: ${{ inputs.working-directory }}
```

Comment:
Did you test this? This will not work for Gradle projects as they use a Gradle plugin for versioning and the version number can be found in a `gradle.properties` file, e.g.: https://github.com/bakdata/streams-bootstrap/blob/master/gradle.properties#L1

Reply:
oh i used a .bumpversion.cfg to test it. you're right it wont work. should i update the bump action by adding a new var for the config ?

Reply:
No, the bumping needs to happen with Gradle and we don't want to change that

Reply:
This is why Gradle still needs to be part of this action, but can be removed in the [java-gradle-release-github](https://github.com/bakdata/ci-templates/blob/main/actions/java-gradle-release-github/action.yaml#L51-L57) action when you remove the changelog there

Reply:
so you want to just change the [changelog step](https://github.com/bakdata/ci-templates/blob/main/actions/java-gradle-release/action.yaml#L89-L95)?

Reply:
In this action yes

Reply:
ok makes sense